types/lib/entities.d.ts

```diff
@@ -2,7 +2,11 @@ export interface Credentials {
     key: string;
     secret: string;
 }
-export interface Account {
+/**
+ * The account information with unparsed types, exactly as Alpaca provides it.
+ * We encourage you to use the Account interface, which has many of these fields parsed.
+ */
+export interface RawAccount {
```

Comment:
I would prefer for `RawAccount` to not be included in the generated type definitions, but I don't know how to exclude it. Should I keep trying or can we live with this?